packages/presentation/lib/src/pages/movie_details_page/widgets/cast_and_crew.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/generated/l10n.dart';
+import 'package:presentation/src/common_widgets/image_error_widget.dart';
```

Comment:
check comma file

Reply:
I checked commas in this file, and faced cases only like this:
        borderRadius: BorderRadius.circular(Dimens.border_r_25),
or
      padding: EdgeInsets.all(Dimens.padding_15),
I don't want to add commas in cases like that.  This is more compact and readable because of this expression has a size <= line size and have no more than 1 constructor arguments